### ISSUE TITLE
overlap error

### ISSUE THREAD

**Issue by Bioinformatics-rookie** [marbl/canu#2265]
I get the following error when running canu
```
-- Final error rates before starting pipeline:
--
--   genomeSize          -- 350000000
--   errorRate           -- 0.015
--
--   corOvlErrorRate     -- 0.045
--   obtOvlErrorRate     -- 0.045
--   utgOvlErrorRate     -- 0.045
--
--   obtErrorRate        -- 0.045
--
--   cnsErrorRate        -- 0.045
--
--
-- BEGIN ASSEMBLY
--
--
-- 217 overlapper jobs failed:
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000001 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000002 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000003 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000004 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000005 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000006 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000007 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000008 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000009 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000010 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000011 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000012 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000013 FAILED.
--   job /ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/001/000014 FAILED.

================================================================================
Don't panic, but a mostly harmless error occurred and canu failed.

canu failed with 'failed to overlap.  Made 2 attempts, jobs still failed'.
```

**Comment by skoren** [comment]
It is suspicious that all the jobs are failing, what's in the logs for one of the failed jobs (`/ds3200_1/users_root/zhoushizhao/Q31-1rawData/00.rawData/canu/350/unitigging/1-overlapper/*.out`). Also, post the canu command and version you're using along with the full report file from the run so far.

**Comment by Bioinformatics-rookie** [comment]
[overlap.1.out.txt](https://github.com/marbl/canu/files/12825831/overlap.1.out.txt)

canu version 1.4-1

command: canu -assemble -p 31-1 -d 350 maxMemory=50G maxThreads=24 genomeSize=350000000 -pacbio-raw ../CCS/ccs.fastq.gz

all files: 
.
├── canu-logs
│   ├── 1696487393_node13_1590546_canu
│   ├── 1696487393_node13_1590674_gatekeeperCreate
│   ├── 1696487800_node13_1591012_gatekeeperDumpMetaData
·········
├── canu-scripts
│   ├── canu.01.out
│   ├── canu.01.sh
│   ├── canu.02.out
│   └── canu.02.sh
├── unitigging
│   ├── 0-mercounts
│   │   ├── 31-1.ms22.estMerThresh.err
│   │   ├── 31-1.ms22.estMerThresh.out
│   │   ├── 31-1.ms22.frequentMers.fasta
│   │   ├── 31-1.ms22.histogram
│   │   ├── 31-1.ms22.histogram.info
│   │   ├── 31-1.ms22.histogram.lg.gp
│   │   ├── 31-1.ms22.histogram.lg.svg
│   │   ├── 31-1.ms22.histogram.sm.gp
│   │   ├── 31-1.ms22.histogram.sm.svg
│   │   ├── meryl.1.out
│   │   ├── meryl.jobSubmit.sh
│   │   └── meryl.sh
│   ├── 1-overlapper
│   │   ├── 31-1.partition.err
│   │   ├── 31-1.partition.ovlbat
│   │   ├── 31-1.partition.ovljob
│   │   ├── 31-1.partition.ovlopt
│   │   ├── overlap.100.out
·····
│   │   ├── overlap.9.out
│   │   ├── overlap.jobSubmit.sh
│   │   └── overlap.sh
│   ├── 31-1.gkpStore
│   │   ├── blobs
│   │   ├── errorLog
│   │   ├── info
│   │   ├── info.txt
│   │   ├── libraries
│   │   ├── libraries.txt
│   │   ├── load.dat
│   │   ├── readlengthhistogram.txt
│   │   ├── readlengths.gp
│   │   ├── readlengths.lg.svg
│   │   ├── readlengths.sm.svg
│   │   ├── readlengths.txt
│   │   ├── readNames.txt
│   │   ├── reads
│   │   └── reads.txt
│   ├── 31-1.gkpStore.err
│   └── 31-1.gkpStore.gkp
├── unitigging.html
└── unitigging.html.files
    ├── 31-1.ms22.histogram.lg.svg
    ├── 31-1.ms22.histogram.sm.svg
    ├── readlengths.lg.svg
    └── readlengths.sm.svg

**Comment by brianwalenz** [comment]
Version 1.4 is *verrry* old.  There were apparently many issues with PBS support that were fixed in v1.5 (see 'Changes' in https://github.com/marbl/canu/releases/tag/v1.5).

I recommend upgrading to the latest release.  You can probably reuse the corrected and trimmed reads, but since v1.4 is so old, I'd do a full restart.

**Comment by Bioinformatics-rookie** [comment]
Thanks, I tried the latest version of canu
